roles/etcd_common/tasks/backup.yml

```diff
@@ -39,7 +38,7 @@
     msg: >
       {{ l_etcd_disk_usage.stdout }} Kb disk space required for etcd backup,
       {{ l_avail_disk.stdout }} Kb available.
-  when: (r_etcd_common_embedded_etcd | bool) and (l_etcd_disk_usage.stdout|int > l_avail_disk.stdout|int)
+  when: l_etcd_disk_usage.stdout|int*2 > l_avail_disk.stdout|int
```

Comment:
I read this as if the usage of etcd is double the amount of what is available then fail .... but that doesn't make sense to me as to why. Shouldn't the etcd usage be under the amount of available space?

Reply:
> Shouldn't the etcd usage be under the amount of available space?

Right, if ($usage * 2) > $free then we need to abort via this fail task.